### PR TITLE
Describe customers workspace controls

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) describes:
- customers workspace controls
- additional columns availability 
- column descriptions

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/customers/customers-all.html

### CODE REVIEW

File: src/customers/customers-all.md [hunk 1]
```diff
@@ -18,3 +18,52 @@ _All Customers_{:.b2b-only}
...
+|Control|Description|
+|--- |--- |
+|Add New Customer|Creates a new customer account.|
+|Search|Initiates a search for customers based on the current filters.|
+|Filters|Defines a set of search parameters used to filter the records that appear in the [grid]({% link stores/admin-grid-controls.md %}).|
+|[Default View]({% link stores/admin-grid-layout.md %})|Determines the default column layout of the grid.|
+|Columns|Determines the selection of [columns]({% link stores/admin-grid-layout.md %}) and their accounts in the grid. The column layout can be changed and saved as a _view_. By default, only some of the columns are included in the grid.|
+|Export|Exports the selected records as a CSV or Excel XML file.|
+
+## Column descriptions
```
Comment (erikmarr) [inline src/customers/customers-all.md:33]: From what I can tell the **Column descriptions** and **Additional Columns Available** match fairly closely to the following existing tables:
- https://docs.magento.com/user-guide/customers/customer-account-manage.html#columns
- https://docs.magento.com/user-guide/customers/customer-account-manage.html#additional-columns
Please make sure the descriptions in your PR match those in the above two links.

Comment (erikmarr) [inline src/customers/customers-all.md:33]:
```suggestion
## Columns
```

File: src/customers/customers-all.md [hunk 2]
```diff
@@ -18,3 +18,52 @@ _All Customers_{:.b2b-only}
...
+|Confirmed Email|Basic trigger email included to customer registration.|
+|Account Created in| Store in wich created customer account.|
+|Date of Birth|Date when customer was birth.|
+|Tax VAT Number|The value-added tax number that is assigned to the customer account by some jurisdictions for tax reporting purposes.|
+|Gender|Gender of the customer.|
+|Company|The company that the customer user account belongs to.|
+|Status|Indicates the current state of the customer account. Options: Active/ Inactive.|
+|Action|View opens the customer in edit mode.|
+
+### Additional Columns Available
```
Comment (erikmarr) [inline src/customers/customers-all.md:58]:
```suggestion
### Additional Columns
```

**Review by erikmarr — CHANGES_REQUESTED** [review]

File: src/customers/customers-all.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/customers-all.md]: Hi @erikmarr, thank you for reviewing!

I have implemented your suggestions, could you please check and approve?

Comment (EvheniiaKolkhiieva) [reply src/customers/customers-all.md]: Done, please check!

File: src/customers/customers-all.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/customers-all.md]: Done, please check!

**Review by erikmarr — APPROVED** [review]

**Comment by erikmarr** [comment]
@EvheniiaKolkhiieva looks good! Thanks for your contribution.

**Comment by magento-engcom-team** [comment]
@erikmarr the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1218`.

**Pull request merged**